### PR TITLE
Adding Logs Sink Connector and Global Sink Timeouts

### PR DESCRIPTION
**Pull request by maju6406**
* Adding new Logs Sink Connector for sending Kafka messages to New Relic Logs.
* Added new `nr.timeout` parameter for all connectors. Default is 2 seconds.
* Updating Telemetry SDK version from 0.6.0 to 0.12.0
* Added simple GitHub action that runs tests on commits and PR's

### CODE REVIEW

**Comment by maju6406** [comment]
Addressing Issue #22

**Review by rahulbats — COMMENTED** [review]
the timestamp logic is nicer with the new logs code will it be ported over to metrics and sink

File: newrelic-kafka-connector/README.md [hunk 1]
```diff
@@ -69,21 +69,44 @@ common message formats such as Prometheus
...
+   "connector.class": "com.newrelic.telemetry.logs.TelemetryLogsSinkConnector",
+   "value.converter":"com.newrelic.telemetry.logs.LogsConverter",
+   "value.converter.schemas.enable": false,
+   "topics": "nrlogs",
+   "api.key": "<NEW_RELIC_API_KEY>"
+   }
+  }
+    ```
+
+#### Timestamps in Logs
```
Comment (rahulbats) [inline newrelic-kafka-connector/README.md:89]: i really like this timestamp check but not sure if this is ported over to metrics and events

**Review by rahulbats — COMMENTED** [review]
looks good

File: newrelic-kafka-connector/README.md (hunk 1 above)
Comment (maju6406) [reply newrelic-kafka-connector/README.md:89]: I tried to keep the changes to Events and Metrics to a minimum. The only thing I added was a timeout, but that was more of a necessity. I could add that to Events and Metrics in a separate PR.

**Comment by maju6406** [comment]
> looks good

@rahulbats Good to merge?

**Pull request merged**